autoload/lsp/codeaction.vim

```diff
@@ -82,12 +82,13 @@ export def ApplyCodeAction(lspserver: dict<any>, actionlist: list<dict<any>>, qu
 
   var choice: number
 
-  if query =~ '^\d\+'	# digit
-    choice = query->str2nr()
-  elseif query =~ '^/'	# regex
-    choice = 1 + util.Indexof(actions, (i, a) => a.title =~ query[1 : ])
-  elseif query != ''	# literal string
-    choice = 1 + util.Indexof(actions, (i, a) => a.title[0 : query->len() - 1] == query)
+  var query_ = query->trim()
```

Comment:
Instead of introducing another "query_" variable, can you reuse query itself here?